data_validation/secret_manger.py

```diff
@@ -0,0 +1,53 @@
+class SecretMangerBuilder:
```

Comment:
Let's add the Google copyright at the top of this - you can just copy/paste from the other files

Reply:
Also the file is named "secret_manger.py" rather than "secret_manager.py".

Reply:
adding copyright headers